### ISSUE TITLE
Possible EventEmitter memory leak detected.

### ISSUE THREAD

**Issue by decoderid** [andresusanto/easydl#10]
how to solve this issue?

## ERROR
(node:20940) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 error listeners added to [WriteStream]. Use emitter.setMaxListeners() to increase limit
(Use `node --trace-warnings ...` to show where the warning was created)

```js
import EasyDl from 'easydl'

const main = async () => {
	try {
		const completed = await new EasyDl('https://code.jquery.com/jquery-3.6.4.js', 'test/a.js', { connections: 5, maxRetry: 5 }).wait()
		console.log('Completed:', completed)
	} catch (err) {
		console.log('ERROR:', err)
	}
}

main()
```

**Comment by ptheofan** [comment]
Also getting the same issue.

**Comment by shakilofficial0** [comment]
Same

**Comment by ahm750** [comment]
@andresusanto Any update on this issue? This download library is gold and the best one by far, but the only thing that's holding me back from using it in production is the memory leak warning.

**Comment by shakilofficial0** [comment]
bump

**Comment by andresusanto** [comment]
thanks for raising the issue, I'll have a look in the coming week

**Comment by ahm750** [comment]
> thanks for raising the issue, I'll have a look in the coming week

Appreciate it!

**Comment by andresusanto** [comment]
Fixed in `1.1.0`
